### PR TITLE
Update link.js

### PR DESCRIPTION
**Pull request by suhaotian**
Then we can write like this
```js
<Link href="/about" className="about" style={{color: 'red'}}>About</Link>
```

### CODE REVIEW

**Review by arunoda — CHANGES_REQUESTED** [review]
Yep. this is good.
You need to fix some lint issues.
Run this:

```
npm run lint -- --fix
```

**Comment by suhaotian** [comment]
@arunoda
Nothing happen
image

File: lib/link.js [hunk 1]
```diff
@@ -44,17 +44,19 @@ export default class Link extends Component {
   }
 
   render () {
-    const children = Children.map(this.props.children, (child) => {
+    let {children, href, as, ...rest} = this.props
+    children = Children.map(children, (child) => {
       const props = {
-        onClick: this.linkClicked
+        onClick: this.linkClicked,
+        ...rest,
```
Comment (arunoda) [inline lib/link.js]: You need to remove this comma here.

**Review by arunoda — CHANGES_REQUESTED** [review]
Change this.

**Comment by arunoda** [comment]
@suhaotian weird.
Check my above comment.

**Comment by suhaotian** [comment]
@arunoda Hey, does right?

**Comment by nkzawa** [comment]
I believe we don't want to support this. See https://github.com/zeit/next.js/issues/272 cc @rauchg

**Comment by suhaotian** [comment]
@nkzawa ok

**Pull request closed without merging**